src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReaderProvider.cs

```diff
@@ -243,7 +243,7 @@ public void Dispose()
         /// <exception cref="PlatformNotSupportedException">The current platform is big-endian.</exception>
         /// <exception cref="IOException">IO error while reading from the underlying stream.</exception>
         /// <exception cref="ObjectDisposedException">Provider has been disposed.</exception>
-        public unsafe MetadataReader GetMetadataReader(MetadataReaderOptions options = MetadataReaderOptions.Default, MetadataStringDecoder utf8Decoder = null)
+        public unsafe MetadataReader GetMetadataReader(MetadataReaderOptions options = MetadataReaderOptions.ApplyWindowsRuntimeProjections, MetadataStringDecoder utf8Decoder = null)
```

Comment:
There was a source/ref assembly mismatch, so I fixed the source based on the documentation:
https://github.com/dotnet/corefx/blob/16f3eef83d6df4c9dbc0e2992dafbc9553db363d/src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReaderOptions.cs#L15-L20

Reply:
These have the same value, but the intent was to use Default. I think this came up last time.

Reply:
Would this still be considered a breaking change?

Also, looking at the documentation (https://docs.microsoft.com/en-us/dotnet/api/system.reflection.metadata.metadatareaderoptions?view=netcore-2.2), it seems contradictory to use `Default` when the overload takes `MetadataReaderOptions`, which is the case here.
```text
The options that are used when a MetadataReader is obtained via an overload that does not take a MetadataReaderOptions argument.
```

Reply:
I think that comment dates from before any optional params were used or allowed. Default and ApplyWindowsRuntimeProjection have the same value. They are indistinguishable at runtime. It is the default. So this is a styke change onky. 

The comment could use updating. But the idea remains that you get default option when yoy don't pass your own value. 

Mind you, I think maybe the intent was that Default could change to include other options, but that could be breaking, so maybe it wasn't the best idea.

Reply:
It would have been intended to use Default inthe ref sources too. There was no ref assembly when this library was first written. The problem is that the tooling operates on binaries not source so it can't tell which spelling the source used.